adabot/circuitpython_libraries.py

```diff
@@ -230,13 +230,14 @@ def run_library_checks(validators, bundle_submodules, latest_pylint, kw_args):
         int(pr_sort_re.search(pr).group(1)) for pr in lib_insights["open_prs"]
         if pr_sort_re.search(pr) is not None
     ]
-    output_handler(
-        "  * {0} open pull requests (Oldest: {1}, Newest: {2})".format(
-            len(lib_insights["open_prs"]),
-            max(open_pr_days),
-            max((min(open_pr_days), 1)) # ensure the minumum is '1'
+    if len(lib_insights["open_prs"]) != 0:
+        output_handler(
+            "  * {0} open pull requests (Oldest: {1}, Newest: {2})".format(
+                len(lib_insights["open_prs"]),
+                max(open_pr_days),
+                max((min(open_pr_days), 1)) # ensure the minumum is '1'
+            )
```

Comment:
Why was this change necessary?

Reply:
I _think_ its to guard against an exception with `max(open_pr_days)`, when `open_pr_days` is an empty list (zero open PRs). Not a bad guard to have, imo.

Reply:
This was noted in the individual commit message, but I didn't call it out again in the PR summary:
>  circuitpython_libraries.py: Don't error if open_prs is empty
>
> This could occur during my testing, which I restricted manually to a tiny subset of libs.  Otherwise, it is unlikely to occur.